controllers/transactionController.js

```diff
@@ -0,0 +1,98 @@
+//Controller for items
+import Transaction from "../model/schemas/Transaction.js";
+import db from "../model/db.js";
+
+const transactionController = {
+    // The transaction page
+    transactions: function (req, res) {
+        res.render("transactions", {
+            title: "Transactions",
+            styles: ["index.css", "w2ui-overrides.css"],
```

Comment:
Move to `pages` and `general` folder

Reply:
see 0489b59349b88012dbf6d15c8f61abe8578eab2e